crates/astria-sequencer-relayer/src/validator.rs

```diff
@@ -13,7 +13,7 @@ use tracing::instrument;
 #[derive(Clone, Debug)]
 pub(crate) struct Validator {
     /// The tendermint validator account address; defined as
-    /// Sha256(verification_key)[..20].
+    /// Sha256(`verification_key`)[..20].
```

Comment:
That's going to look very strange:
```suggestion
    /// `Sha256(verification_key)[..20]`.
```